apps/backoffice-v2/src/pages/MerchantMonitoringBusinessReport/MerchantMonitoringBusinessReport.page.tsx

```
@@ -48,6 +74,8 @@
 import { SidebarInset, SidebarProvider } from '@/common/components/organisms/Sidebar/Sidebar';
 import { MerchantMonitoringReportStatus } from '@/pages/MerchantMonitoring/components/MerchantMonitoringReportStatus/MerchantMonitoringReportStatus';
 import { useMerchantMonitoringBusinessReportLogic } from '@/pages/MerchantMonitoringBusinessReport/hooks/useMerchantMonitoringBusinessReportLogic/useMerchantMonitoringBusinessReportLogic';
+import { MERCHANT_REPORT_STATUSES_MAP } from '@ballerine/common';
```

Biome (1.9.4)

[error] Shouldn't redeclare 'MERCHANT_REPORT_STATUSES_MAP'. Consider to delete it or rename it.

'MERCHANT_REPORT_STATUSES_MAP' is defined here:


(lint/suspicious/noRedeclare)